### PR TITLE
OCPBUGS-3791: remove mention of atomic

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OCPBUGS-3791

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OCPBUGS-3791/installing/index.html#about-rhcos

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@mike-nguyen could you PTAL for QE ack? this PR removes the mention of "Atomic" from "Atomic OSTree", and also includes a glossary addition for OSTree, which is from the upstream docs. lmk if some of it isn't pertinent for OCP, I wasn't sure how much information to include. Thank you!

File: modules/installation-overview.adoc [hunk 1]
```diff
@@ -39,6 +39,6 @@ image::targets-and-dependencies.png[{product-title} installation targets and dep
...
+Every control plane machine in an {product-title} {product-version} cluster must use {op-system}, which includes a critical first-boot provisioning tool called Ignition. This tool enables the cluster to configure the machines. Operating system updates are delivered as an **OSTree** repository that is embedded in a container image that is deployed across the cluster by an Operator. Actual operating system changes are made in-place on each machine as an atomic operation by using **rpm-ostree**. Together, these technologies enable {product-title} to manage the operating system like it manages any other application on the cluster, by in-place upgrades that keep the entire platform up-to-date. These in-place updates can reduce the burden on operations teams.
```
Comment (cgwalters) [inline modules/installation-overview.adoc:42]: I'd prefer that we de-emphasize ostree a bit more.  How about:

> Operating system updates are delivered as a bootable container image (using **OSTree** as a backend) that is deployed across the cluster by the Machine Config Operator.

Comment (jldohmann) [reply modules/installation-overview.adoc:42]: thanks @cgwalters! would this be for version 4.12+?

Comment (cgwalters) [reply modules/installation-overview.adoc:42]: Yep 4.12+

**Comment by jldohmann** [comment]
710bb40..f11c24c implement SME feedback

**Comment by mike-nguyen** [comment]
LGTM

**Pull request merged**